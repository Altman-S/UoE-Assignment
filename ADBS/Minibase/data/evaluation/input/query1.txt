Q(x,y,z) :- R(x, y, z)