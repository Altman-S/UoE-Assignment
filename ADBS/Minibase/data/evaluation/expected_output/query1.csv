1, 9, 'adbs'
2, 7, 'anlp'
4, 2, 'ids'
5, 0, 'sdp'
8, 1, 'mlpr'
8, 9, 'rl'
8, 9, 'ppls'